src/app/global_styles.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { createGlobalStyle, DefaultTheme, useTheme } from "styled-components";
+import {
+    ThemeProvider as MaterialThemeProvider,
+    createTheme as createMaterialTheme,
+    Theme,
+} from "@mui/material";
+import { lightTheme } from "@/app/themes";
+
+const GlobalStyle = createGlobalStyle`    
+    * {
+        box-sizing: border-box;
+        margin: 0;
+    }
+      
+    html, body {
+        height: 100%;
+        width: 100%;
+        font-family: Helvetica, Arial, sans-serif;
+        overflow: hidden;
+    }
+
+    body {
+        color: ${(props) => props.theme.main.foreground[2]};
+        background-color: ${(props) => props.theme.main.background[2]};
+    }
+
+    :root {
+      color-scheme: ${(props) => (props.theme.light ? "light" : "dark")}
+    }
+`;
+
+const materialTheme = (chosenTheme: DefaultTheme): Theme =>
+    createMaterialTheme({
+        palette: {
+            mode: chosenTheme === lightTheme ? "light" : "dark",
```

Comment:
not a massive fan of importing `lightTheme` just for comparison - can this be `currentTheme.light` or something?